packages/routes/package.json

```diff
@@ -42,9 +42,9 @@
     "mock-fs": "^4.5.0"
   },
   "dependencies": {
-    "@seagull/commands-fs": "^3.4.2",
-    "@seagull/http": "^3.4.2",
-    "@seagull/libraries": "^3.4.2",
+    "@seagull/commands-fs": "3.4.2",
```

Comment:
Is there a reason to prevent future updates from happening?

Reply:
fixed